languages/translations/de.json

```diff
@@ -84,23 +84,34 @@
         },
         "continue": {
             "default": {
-                "default": "{modifier} weiterfahren",
-                "name": "{modifier} weiterfahren auf {way_name}",
-                "destination": "{modifier} weiterfahren Richtung {destination}"
+                "default": "{modifier} abbiegen",
+                "name": "Turn {modifier} to stay on {way_name}",
+                "destination": "{modifier} abbiegen Richtung {destination}",
+                "exit": "{modifier} abbiegen auf {way_name}"
+            },
+            "sharp left": {
+                "default": "Scharf links",
+                "name": "Make a sharp left to stay on {way_name}",
+                "destination": "Scharf links Richtung {destination}"
```

Comment:
Here and below: this feels a bit too short, it's basically

"Sharp right towards"